metrics-httpclient/src/main/java/com/codahale/metrics/httpclient/InstrumentedHttpClientConnectionManager.java

```diff
@@ -114,4 +82,76 @@ public void shutdown() {
         metricsRegistry.remove(name(HttpClientConnectionManager.class, name, "max-connections"));
         metricsRegistry.remove(name(HttpClientConnectionManager.class, name, "pending-connections"));
     }
+
+    public static Builder builder(final MetricRegistry metricsRegistry) {
+        return new Builder().metricsRegistry(metricsRegistry);
+    }
+
+    public static class Builder {
```

Comment:
Could you please write a unit test for `Builder`? Just with specifying a `MetricsRegistry` and a name as you described in the issue.

Reply:
Sure.